### ISSUE TITLE
OKLAB Colorspace - for better perceptual colorspace experience

### ISSUE THREAD

**Issue by TheBagarius** [juxtopposed/realtimecolors#93]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the problem

Adding support for better perceptual colorspaces will enable more people to work with this tool more intuitively rather than limiting to designers who have learned to work the limitations of typical colorspaces such as HSL.

### Solution

[OKLAB Whitepaper](https://bottosson.github.io/posts/oklab/)

[MDN Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/oklab)

[Javascript Implementation](https://github.com/Evercoder/culori) as linked in the whitepaper

[TypeScript Implimentation](https://github.com/thi-ng/umbrella/tree/develop/packages/color) as linked in the whitepaper

### Alternatives

OKLCH - already described in issue #37 

### Additional context

[THIS](https://atmos.style/blog/lch-vs-oklch) article explains the difference between OKLAB and OKLCH

**Comment by juxtopposed** [comment]
Absolutely. This feature is planned and should be released in the upcoming updates.

**Comment by TheBagarius** [comment]
I saw the new update has the ability to export OKLAB and OKLCH. AND I LOVE THAT.

Can I request for those color spaces to also be available in the colour pickers?
In my opinion that is where they really flex their proverbial muscles.

Try out the massively overkill [OKLCH colour picker ](https://oklch.com/)to get a feel for what I mean. I find myself using this tool when picking colours more than anything else ever since I've learned about it.
